src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -67,11 +65,14 @@ public function theTaxonAndTaxonAreExcludedFromShowingTheLowestPriceOfDiscounted
     }
 
     /**
-     * @Given the store operates on a single channel in "United States"
+     * @Given the store operates on a single channel in :channelName
```

Comment:
I guess it either should become
```suggestion
     * @Given the store operates on a single channel in the "United States" named :channelName
```
or US should be removed from method name and `$this->unitedStatesChannelFactory` should be replaced with `$this->defaultChannelFactory`

Reply:
I reverted these changes as I overlooked that it uses a specific factory